src/util/deprecated.js

```diff
@@ -0,0 +1,5 @@
+export default function deprecated (message) {
+  // TODO start throwing in a future version
```

Comment:
I would say instead that we would think about just removing the deprecation message along with the functionality. New, breaking code shouldn't contain errors for legacy code, I don't think.